staging/src/k8s.io/apiserver/pkg/storage/cacher/cacher.go

```diff
@@ -1004,6 +1010,7 @@ func (c *Cacher) isStopped() bool {
 
 // Stop implements the graceful termination.
 func (c *Cacher) Stop() {
+	c.ready.stop()
```

Comment:
should this be inside `c.stopLock.Lock()` and after the `if c.stopped` check to avoid calling twice?